manifest.toml

```diff
@@ -0,0 +1,104 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "jeedom"
+name = "Jeedom"
+description.en = "Home automation application"
+description.fr = "Application de domotique"
+
+version = "4.1.19~ynh5"
+
+maintainers = []
+
+[upstream]
+license = "GPL-2.0-or-later"
+website = "https://www.jeedom.com"
+admindoc = "https://doc.jeedom.com"
+code = "https://github.com/jeedom/core"
+cpe = "cpe:2.3:a:jeedom:jeedom"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = false
+ldap = true
+sso = true
+disk = "200M"
+ram.build = "200M"
+ram.runtime = "200M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/jeedom"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.admin]
+    type = "user"
+
+    [install.sudo]
+    ask.en = "Do you agree giving sudo rights to the Jeedom user? This is required for most Jeedom features to work. However, attackers could use vulnerabilities in Jeedom to take control of your server."
+    ask.fr = "Acceptez-vous de donner les droits sudo à l'utilisateur Jeedom ? Ces droits sont nécessaires au bon fonctionnement de Jeedom. Toutefois, des attaquants pourraient exploiter des failles de Jeedom pour prendre le controle de votre serveur."
+    type = "boolean"
+    default = true
+
+[resources]
+    [resources.sources.main]
+    url = "https://github.com/jeedom/core/archive/refs/tags/4.1.19.tar.gz"
+    sha256 = "be50f67286d1533c93584f3653d4669b36d649033ad1750342b704f9a824476a"
+
+    autoupdate.strategy = "latest_github_tag"
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = [
+        "ntp", "ca-certificates", "unzip", "curl", "cron", "locate", "tar",
+        "telnet", "wget", "logrotate", "fail2ban", "dos2unix", "ntpdate", "htop",
+        "iotop", "vim", "iftop", "smbclient", "git", "python3", "python3-pip",
+        "software-properties-common", "libexpat1", "ssl-cert", "apt-transport-https",
+        "xvfb", "cutycapt", "xauth", "duplicity", "at", "ffmpeg",
```

Comment:
Uuuuuuh wokay, does it really needs vim, smbclient, ssl-cert, apt-transport-https and telnet ? x_x

Reply:
```suggestion
        "unzip", "cron", "locate", "tar",
        "logrotate", "dos2unix", "python3", "python3-pip", "libexpat1",
        "xvfb", "cutycapt", "xauth", "duplicity", "at", "ffmpeg",
```